docs/config.rst

```diff
@@ -39,14 +39,38 @@ compiles your project contracts.
         "backend": {
           "class": "populus.compilation.backends.SolcCombinedJSONBackend",
           "settings": {
-              "optimize": true,
-              "optimize_runs": 100,
-              "output_values": ["abi", "bin", "bin-runtime", "devdoc", "metadata", "userdoc"]
-          }
+              "stdin": {
+                "optimizer": {
+                  "enabled": true,
+                  "runs": 500
+                },
+                "outputSelection": {
+                  "*": {
+                    "*": [
+                      "abi",
+                      "metadata",
+                      "evm.bytecode",
+                      "evm.deployedBytecode"
+                    ]
+                  }
+                }
+              },
+              "command_line_options": {
+                "allow_paths": "/"
```

Comment:
This is not a good default to provide.  This is effectively bypassing all of the security that `--allowed-paths` is supposed to impose and I'd prefer not to have this type of thing in the documentation.

I'd suggest replacing this with a pseudo path like `/path-to-your-external-solidity-files`